bin/create-docs-pr.js

```diff
@@ -108,7 +113,7 @@ async function getReleaseNotes(version, releaseNotesFile) {
   const versionChangeLog = sections.find((section) => section.startsWith(version))
   // e.g. v7.1.2 (2021-02-24)\n\n
   const headingRegex = /^v\d+\.\d+\.\d+ \((\d{4}-\d{2}-\d{2})\)\s+/
-  const body = versionChangeLog.replace(headingRegex, '')
+  const body = versionChangeLog.replace(headingRegex, '') + SUPPORT_STATEMENT
```

Comment:
If we got to a point we could trigger these off of GitHub releases and use those notes, we could remove the need to duplicate adding this. Not a big deal currently.

Reply:
I couldn't figure out how to do it since the docs PR requires "owernship", to be assigned to a human on our team that will interface with the docs team. Maybe if we added our credentials to the github action that creates the release, that way github actions can generate a PR with the name of the person who triggered the release?